syntax = "proto3";

message EventInitialHand {
	uint32 count = 1;
	repeated string codes = 2;
}
